packages/client/src/helpers/PeerConnectionHandler.ts

```diff
@@ -0,0 +1,86 @@
+import { detectAudioLevels } from './detect-audio-levels';
+import { isReactNative } from './platforms';
+import { SoundStateChangeHandler } from './sound-detector';
+
+export class PeerConnectionHandler {
```

Comment:
I would name this class differently - `RNSpeechDetector` - it shows its purpose. PeerConnections are just an implementation detail here and I wouldn't leak them to the outer world.